src/shared/browser/chrome.ts

```diff
@@ -0,0 +1,65 @@
+import { IBrowser } from './types'
+
+export const chromeBrowser: IBrowser = {
+  storage: {
+    local: {
+      get: (key, defaultValue) => {
+        return new Promise((resolve) => {
+          chrome.storage.local.get([key], (storage) => {
+            try {
+              const json = JSON.parse(storage[key])
+              resolve(json)
+            } catch (error) {
+              resolve(defaultValue)
+            }
+          })
+        })
+      },
+
+      set: (key, value) => {
+        return new Promise((resolve) => {
+          let stringifiedValue = ''
+          try {
+            stringifiedValue = JSON.stringify(value)
+            chrome.storage.local
+              .set({
+                [key]: stringifiedValue,
+              })
+              .then(resolve)
+          } catch (error) {
+            resolve()
```

Comment:
I think we shouldn't simply resolve if there was an error. We need to at least show user an error that something went wrong. let's add `reject()` with meaningful error here